core/src/Streamly/Internal/Data/Stream/Transform.hs

```diff
@@ -262,6 +263,10 @@ foldrT f z s = D.foldrT f z (toStreamD s)
 mapM :: Monad m => (a -> m b) -> Stream m a -> Stream m b
 mapM f m = fromStreamK $ D.toStreamK $ D.mapM f $ toStreamD m
 
+{-# INLINE map #-}
+map :: Monad m => (a -> b) -> Stream m a -> Stream m b
+map f = mapM $ return . f
+
```

Comment:
I think we discussed this before. I deliberately removed `map`. We can use `fmap` wherever this is needed. Please remove this.